docs/dependencies.md

```diff
@@ -0,0 +1,27 @@
+# Dependency Upgrades
```

Comment:
Nothing links to this page, it will be impossible to find. Also I'm not sure it should be a separate page. We could link to the PR?